server/src/tenant/query-builder/factories/composite-field-alias.factory.ts

```diff
@@ -95,10 +95,20 @@ export class CompositeFieldAliasFactory {
       }
     `;
     }
+    let relationAlias = fieldKey;
+
+    // For one to one relations, pg_graphql use the targetTableName on the side that is not storing the foreign key
+    // so we need to alias it to the field key
+    if (
+      relationMetadata.relationType === RelationMetadataType.ONE_TO_ONE &&
+      relationDirection === RelationDirection.FROM
+    ) {
+      relationAlias = `${fieldKey}: ${referencedObjectMetadata.targetTableName}`;
```

Comment:
I thought we shouldn't use targetTableName as an alias because we can have multiple fields using the same relation, does it work?

Reply:
@Weiko Actually on a `one-to-one` relation I don't think we can as the id should be unique ?
Actually on `pg_graphql` I can rename it with `comment` but I not sure it will allow to use 2 relations on the same entity.